core/block/editor/state/change.go

```diff
@@ -823,6 +845,34 @@ func (s *State) makeNotificationChanges() []*pb.ChangeContent {
 	return changes
 }
 
+func (s *State) makeDeviceInfoChanges() []*pb.ChangeContent {
+	if s.parent == nil && len(s.parent.ListDevices()) == 0 {
```

Comment:
1. Nil deref here(in case s.parent is nil)
2. Also for the first change should it work fine if the parent is nil but we have devices in the current state?

Reply:
Fixed and add test